libs/network/esp8266/jswrap_esp8266_network.c

```diff
@@ -765,18 +787,31 @@ JsVar *jswrap_ESP8266_wifi_getStatus() {
 * * ip - IP address
 * * netmask - The interface netmask
 * * gw - The network gateway
+* * mac - The MAC address
 */
 JsVar *jswrap_ESP8266_wifi_getIP() {
   os_printf("> jswrap_ESP8266_wifi_getIP\n");
-  struct ip_info info;
-  wifi_get_ip_info(0, &info);
-
-  JsVar *jsIpInfo = jspNewObject(NULL, "Restart");
```

Comment:
I'm not sure this is doing what you want at all (or if it is, it's accidental). Just use `= jsvNewWithFlags(JSV_OBJECT)` like everything else does.